.env.example

```
@@ -0,0 +1,16 @@
+# Database
+DB_HOST=dcts-mariadb
+DB_USER=dcts
+DB_PASS=change_me_in_production
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASS key should go before the DB_USER key

(UnorderedKey)